### DISCUSSION TITLE
msg2 RACH failures seen during UE attach

### DISCUSSION THREAD

**Discussion by ven457** [srsran/srsRAN_Project#680] [General Help]
Hello srsRAN team,
We are trying to perform UE attach with srsRAN + FDD O-RU in n71 band. During RACH procedure, we see RACH attempt failed to msg2 timer expiry. can you please audit our test setup configuration and suggest what could have gone wrong here.? 

**srsRAN log:**
[gnb.log](https://github.com/user-attachments/files/15884947/gnb.log)
**gNB config file:**
[04_low_level_endpoints_carriers_v10.txt](https://github.com/user-attachments/files/15885101/04_low_level_endpoints_carriers_v10.txt)
**RU config file:**
[RU_n71_10mhz_4x4_v10.txt](https://github.com/user-attachments/files/15885116/RU_n71_10mhz_4x4_v10.txt)
**PCAP:**
[srsran_msg2fail.pcap.gz](https://github.com/user-attachments/files/15884948/srsran_msg2fail.pcap.gz)

**Comment by ven457** [comment]
Hi srsRAN team,
can you please check the attached configurations to check why msg2 failures are seen?

**Reply by ismagom** [reply]
Hi @ven457, the community will help when they can. If you need SRS team support in a short timeframe I suggest you contact sales@srs.io.

Thanks

**Reply by ven457** [reply]
Thanks, i will write to **sales@srs.io**
